bitrepository-core/src/test/java/org/bitrepository/protocol/utils/MessageDataTypeValidatorTest.java

```diff
@@ -51,7 +52,11 @@ public void validateChecksumDataForFileInvalidChecksumSpecTest() {
         ChecksumDataForFileTYPE noChecksumSpec = new ChecksumDataForFileTYPE();
         ChecksumSpecTYPE checksumTypeSpec = new ChecksumSpecTYPE();
         noChecksumSpec.setChecksumSpec(checksumTypeSpec);
-        noChecksumSpec.setChecksumValue(Base16Utils.encodeBase16("abab"));
+        try {
+            noChecksumSpec.setChecksumValue(Base16Utils.encodeBase16("abab"));
+        } catch (DecoderException e) {
+            System.err.println(e.getMessage());
```

Comment:
For all these tests (also talking about the other test files here) catching DecoderException is there a reason for not just marking the methods as throwing DecoderException? You did it for some of the tests, so I'm just curious why not do it for all?

Reply:
Must've been a missclick